### DISCUSSION TITLE
Getting error regarding setcookie() after updating Cloudlog

### DISCUSSION THREAD

**Discussion by sebket** [magicbug/Cloudlog#2565] [Q&A]
Hi everyone, I have just updated my Cloudlog installationto the current version (as always just by copying the new files over the old ones).
Now, after I've entered my credentials into the login window, I get the follwoing error message:

An uncaught Exception was encountered
Type: TypeError
Message: setcookie(): Argument #3 ($expires_or_options) must be of type array|int, float given
Filename: D:\wamp\www\cloudlog\system\core\Input.php
Line Number: 351

If then I edit the URL in the browser manually to point to the index.php Cloudlog opens, I am logged in and it seems to work normally.

What can I do to be able to login normally again?

Thanks and 73,
Sebastian DL2KSE

**Comment by phl0** [comment]
Can you please have a look at the developer console to see which cookie is causing the issues?

**Comment by sebket** [comment]
Thanks for your reply.
It seems the problem was a too big step in versions.
I wanted to upgrade from 2.33 straight to the newest version.
Upgrading to 2.4 first as an intermediate step resolved the issue!


Von: Florian (DF2ET) ***@***.***
Gesendet: Sonntag, 15. Oktober 2023 14:09
An: magicbug/Cloudlog ***@***.***>
Cc: Sebastian Kettner ***@***.***>; Author ***@***.***>
Betreff: Re: [magicbug/Cloudlog] Getting error regarding setcookie() after updating Cloudlog (Discussion #2565)


Can you please have a look at the developer console to see which cookie is causing the issues?

**Comment by Matt-BW** [comment]
Did you get this figured out? I have this problem with a fresh install.

Many thanks,
Matt.

**Comment by sebket** [comment]
Hi Matt,

did you really perform a completely fresh install, without trying to restore any data from earlier installations?

Like I've said, in my case the problem was that the upgrade I had intended apparently involved a too big step in versions. After I had changed that and stayed within the same major release number, the problems ceased.

What kind of platform are you  using to run your Cloudlog installation on?

Best 73,
Sebastian
